experimental/PropertyDDS/packages/property-common/src/chronometer.ts

```diff
@@ -125,6 +125,7 @@ let impl:
 if (typeof process !== "undefined" && typeof process.hrtime !== "undefined") {
 	impl = implementations.node;
 } else if (
+	// eslint-disable-next-line @typescript-eslint/prefer-optional-chain -- optional chain is not logically equivalent in this case
 	typeof window !== "undefined" &&
 	typeof window.performance !== "undefined" &&
 	typeof window.performance.now !== "undefined"
```

Comment:
Hmm it's not? `(typeof window?.performance?.now) !== "undefined"` wouldn't produce the same output?

Reply:
I could have sworn there were test failures, but I'll check again.

Reply:
This one didn't cause a test failure, but another one in this project did. I added a comment.